salt/modules/pip.py

```diff
@@ -172,20 +172,7 @@ def _search_paths(*basedirs):
             # If the python binary was passed, return it
             if 'python' in os.path.basename(bin_env):
                 return [os.path.normpath(bin_env), '-m', 'pip']
-            # Try to find the python binary based on the location of pip in a
-            # virtual environment, should be relative
-            if 'pip' in os.path.basename(bin_env):
-                # Look in the same directory as the pip binary, and also its
-                # parent directories.
-                pip_dirname = os.path.dirname(bin_env)
-                pip_parent_dir = os.path.dirname(pip_dirname)
-                for bin_path in _search_paths(pip_dirname, pip_parent_dir):
-                    if os.path.isfile(bin_path):
-                        logger.debug('pip: Found python binary: %s', bin_path)
-                        return [os.path.normpath(bin_path), '-m', 'pip']
-
-            # Couldn't find python, use the passed pip binary
-            # This has the limitation of being unable to update pip itself
+            # We have been passed a pip binary, use the pip binary.
             return [os.path.normpath(bin_env)]
```

Comment:
I think we need to just add this line and keep all the stuff above. I believe it's needed to find pip in Windows.